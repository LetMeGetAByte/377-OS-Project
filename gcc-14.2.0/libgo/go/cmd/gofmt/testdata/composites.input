version https://git-lfs.github.com/spec/v1
oid sha256:e65667eeb8867fe38cb2422cc6da079690ee87a43e3411a77d4daffe6d5679e6
size 3236
