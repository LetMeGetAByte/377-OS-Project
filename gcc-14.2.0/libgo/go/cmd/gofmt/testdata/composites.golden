version https://git-lfs.github.com/spec/v1
oid sha256:51a254ac644d1cfec23acf85759ac34bb15e226b2871e6e150ad40de7f3db0e5
size 2596
